src/material/tree/node.ts

```diff
@@ -87,7 +87,7 @@ export class MatTreeNodeDef<T> extends CdkTreeNodeDef<T> {
   exportAs: 'matNestedTreeNode',
   host: {
     '[attr.aria-expanded]': 'isExpanded',
-    '[attr.role]': 'role',
+    '[attr.role]': `'treeitem'`,
```

Comment:
@annieyw we should leave this part how it was and change the input itself to always be `treeitem`. I suspect this will also require updating the tests